source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -52,6 +52,7 @@
     UnsignedInteger,
     DateTimeFormat,
     PositiveFloat,
+    RangeFloat,
```

Comment:
How about
```suggestion
   SpeedRange,
```

Reply:
Thanks SpeedRange is better than RangeFloat for this case !!
Commit upcoming

Reply:
For me RangeFloat is more generic and could be used everywhere Non?

Reply:
We could have a Generic FloatRange and a specific one to avoid having a big map. 
SpeedRange could contains a simple map with all speed limitation only ?